### PR TITLE
add membership policy

### PR DESCRIPTION
**Pull request by BillWagner**
Post the membership policy (originally adopted in November 2020) on the website.

[Staging site link to new page](https://jolly-grass-05beb181e-1053.westus2.azurestaticapps.net/about/member-policy)

### CODE REVIEW

**Comment by isaacrlevin** [comment]
@BillWagner 
Can you clean up the CSS? Not sure how Statiq handles it, you might need to change this from markdown to css

File: input/about/member-policy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: .NET Foundation Membership Policy
+layout: _BasicPageLayout
+---
+
+ <!-- start -->
+<section class="page-section page-section">
```
Comment (Arhell) [inline input/about/member-policy.md:7]: @BillWagner 
You can remove all html, it is superfluous here
https://jolly-grass-05beb181e-1053.westus2.azurestaticapps.net/about/member-policy

Comment (BillWagner) [reply input/about/member-policy.md:7]: @isaacrlevin Can you advise here? Adding this fixed the CSS, but it does seem to be duplicating the divs used for the bootstrap grid.

**Review by rprouse — APPROVED** [review]
Approved other than the currently open comments.

File: input/about/member-policy.md (hunk 1 above)
Comment (isaacrlevin) [reply input/about/member-policy.md:7]: The current version looks fine. Merging in

**Pull request merged**